demos/python_demos/rag_chatbot/README.md

```diff
@@ -0,0 +1,163 @@
+
+Script to download LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model {all-mpnet-base-v2,text2vec-large-chinese}
+                        Select the LLM model out of supported list
+
+python download_embedding_model.py --model all-mpnet-base-v2
+
+```
+The model will appear in `./all-mpnet-base-v2` directory.
+
+## Download test documents for knowledge base
+We will use single CSV file with [FIFA World Cups information](https://raw.githubusercontent.com/rangelak/fifa-stats/master/data/fifa-world-cup.csv).
+```bash
+wget -P documents/ https://raw.githubusercontent.com/rangelak/fifa-stats/master/data/fifa-world-cup.csv
+```
+
+## Deploy OpenVINO Model Server with Python Calculator
+
+Mount the `./documents` directory with knowledge base.
+Mount the `./llm_model` directory with the LLM model.  
+Mount the `./embed_model` directory with the document embedding model.  
+Mount the servable directory which contains:
+- python scripts which are required for execution and use [Hugging Face](https://huggingface.co/) utilities with [optimum-intel](https://github.com/huggingface/optimum-intel) acceleration.
+- `config.json` - which defines which servables should be loaded
+- `graph.pbtxt` - which defines MediaPipe graph containing python calculator
+
+```bash
+docker run -it --rm -p 9099:9099 -v ${PWD}/servable_stream:/workspace -v ${PWD}/${SELECTED_MODEL}:/llm_model \
+-v ${PWD}/all-mpnet-base-v2:/embed_model -v ${PWD}/documents:/documents -e SELECTED_MODEL=${SELECTED_MODEL} \
+dk_py --config_path /workspace/config.json --port 9099
+```
+
+You may deploy the compressed model(s) by simply changing the model path mounted to the container. For example, to deploy the 8-bit weight compressed model:
+```bash
+docker run -d --rm -p 9000:9000 -v ${PWD}/servable_stream:/workspace -v ${PWD}/${SELECTED_MODEL}_INT8_compressed_weights:/llm_model \
+-v ${PWD}/all-mpnet-base-v2:/embed_model -v ${PWD}/documents:/documents -e SELECTED_MODEL=${SELECTED_MODEL} \
+registry.connect.redhat.com/intel/openvino-model-server:py --config_path /workspace/config.json --port 9000
+```
+> **NOTE** Check the Docker container logs to confirm that the model is loaded before sending requests from a client. Depending on the model and hardware it might take a few seconds or several minutes.
+
+> **Note** If order to run the inference load on Intel GPU instead of CPU, just pass the extra parameters to the docker run `--device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render*)`.
+It will pass the GPU device to the container and set the correct group security context.
+
+## Run gradio client with gRPC streaming
+
+Install gradio and dependencies:
+```bash
+pip install -r client_requirements.txt
+```
+Start the gradio web server:
+```bash
+python3 app.py --web_url localhost:9001 --ovms_url localhost:9000
```

Comment:
OVMS run command above has the port configured for 9099, accordingly here ovms_url has to be localhost:9099 instead of 9000

Reply:
Also, when running behind the proxy, all_proxy and ALL_PROXY have to be unset before running gradio client

Reply:
Fixed!